config/standard_mapping.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
It looks like this mapping file is currently only used for testing. If there's a current common use case for it IRL, I would rename it to match the use case (standard mapping from what?). If not, I would move it to the test/fixtures directory.